<!DOCTYPE html>
<html lang="fr">

<head>
  <title>
  Recherches et Projets ¬∑ Naser Poursalavati
</title>
  <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="color-scheme" content="light dark">




<meta name="author" content="Naser Poursalavati">
<meta name="description" content="D√©couvrez mon parcours de recherche : explorez mes projets acad√©miques, travaux publi√©s et projets en cours sur cette page :

Publications
Projets
Th√®ses
Ateliers, S√©minaires, Pr√©sentations
">
<meta name="keywords" content="bioinformatique, virologie, blog, d√©veloppeur, personnel">

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Recherches et Projets"/>
<meta name="twitter:description" content="D√©couvrez mon parcours de recherche : explorez mes projets acad√©miques, travaux publi√©s et projets en cours sur cette page :

Publications
Projets
Th√®ses
Ateliers, S√©minaires, Pr√©sentations
"/>

<meta property="og:title" content="Recherches et Projets" />
<meta property="og:description" content="D√©couvrez mon parcours de recherche : explorez mes projets acad√©miques, travaux publi√©s et projets en cours sur cette page :

Publications
Projets
Th√®ses
Ateliers, S√©minaires, Pr√©sentations
" />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://www.poursalavati.com/fr/research/" /><meta property="article:section" content="" />
<meta property="article:published_time" content="2023-08-02T00:00:00+00:00" />
<meta property="article:modified_time" content="2023-08-02T00:00:00+00:00" /><meta property="og:site_name" content="Naser Poursalavati" />





<link rel="canonical" href="https://www.poursalavati.com/fr/research/">


<link rel="preload" href="/fonts/forkawesome-webfont.woff2?v=1.2.0" as="font" type="font/woff2" crossorigin>


  
  
  <link rel="stylesheet" href="/css/coder.min.ff9f45cc8b376195e95142374f0afa19a06b212f237923a1f1b69a4af1ac7845.css" integrity="sha256-/59FzIs3YZXpUUI3Twr6GaBrIS8jeSOh8baaSvGseEU=" crossorigin="anonymous" media="screen" />






  
    
    
    <link rel="stylesheet" href="/css/coder-dark.min.f6534b0b446b75d9b6ad77a97d43ede2ddaeff1b6e2361fb7198d6f8fcb7f83f.css" integrity="sha256-9lNLC0Rrddm2rXepfUPt4t2u/xtuI2H7cZjW&#43;Py3&#43;D8=" crossorigin="anonymous" media="screen" />
  



 
  
    
    <link rel="stylesheet" href="/css/cos.min.7424865dbfdd1b21393f514a5c7f6c4be6252059e9809db121ad9d1826048836.css" integrity="sha256-dCSGXb/dGyE5P1FKXH9sS&#43;YlIFnpgJ2xIa2dGCYEiDY=" crossorigin="anonymous" media="screen" />
  





<link rel="icon" type="image/png" href="/images/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="/images/favicon-16x16.png" sizes="16x16">

<link rel="apple-touch-icon" href="/images/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon.png">

<link rel="manifest" href="/site.webmanifest">
<link rel="mask-icon" href="/images/safari-pinned-tab.svg" color="#5bbad5">







<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Vazirmatn:wght@300&display=swap" rel="stylesheet">


</head>






<body class="preload-transitions colorscheme-auto">
  
<div class="float-container">
    <a id="dark-mode-toggle" class="colorscheme-toggle">
        <i class="fa fa-adjust fa-fw" aria-hidden="true"></i>
    </a>
</div>


  <main class="wrapper">
    <nav class="navigation">
  <section class="container">
    <a class="navigation-title" href="/fr">
      Naser Poursalavati
    </a>
    
      <input type="checkbox" id="menu-toggle" />
      <label class="menu-button float-right" for="menu-toggle">
        <i class="fa fa-bars fa-fw" aria-hidden="true"></i>
      </label>
      <ul class="navigation-list">
        
          
            <li class="navigation-item">
              <a class="navigation-link" href="/fr/about/">√Ä propos</a>
            </li>
          
            <li class="navigation-item">
              <a class="navigation-link" href="/fr/now/">Maintenant</a>
            </li>
          
            <li class="navigation-item">
              <a class="navigation-link" href="/fr/posts/">Blog</a>
            </li>
          
            <li class="navigation-item">
              <a class="navigation-link" href="/fr/research/">Recherches/Projets</a>
            </li>
          
            <li class="navigation-item">
              <a class="navigation-link" href="/fr/contact/">Contact</a>
            </li>
          
            <li class="navigation-item">
              <a class="navigation-link" href="/fr/cv/">CV</a>
            </li>
          
        
        
          
          
          
            
              
                <li class="navigation-item menu-separator">
                  <span>|</span>
                </li>
                
              
              <li class="navigation-item">
                <a href="/research/">en</a>
              </li>
            
          
            
              
              <li class="navigation-item">
                <a href="/fa/research/">ŸÅÿßÿ±ÿ≥€å</a>
              </li>
            
          
        
      </ul>
    
  </section>
</nav>


    <div class="content">
      
  <section class="container page">
  <article>
    <header>
      <h1 class="title">
        <a class="title-link" href="https://www.poursalavati.com/fr/research/">
          Recherches et Projets
        </a>
      </h1>
    </header>

    <p><em>D√©couvrez mon parcours de recherche : explorez mes projets acad√©miques, travaux publi√©s et projets en cours sur cette page :</em></p>
<ul>
<li><a href="#publications" >Publications</a></li>
<li><a href="#projects" >Projets</a></li>
<li><a href="#theses" >Th√®ses</a></li>
<li><a href="#workshops" >Ateliers, S√©minaires, Pr√©sentations</a></li>
</ul>
<hr>
<h1 id="publications">
  Publications
  <a class="heading-link" href="#publications">
    <i class="fa fa-link" aria-hidden="true" title="Link to heading"></i>
    <span class="sr-only">Link to heading</span>
  </a>
</h1>
<p><strong>2023 - <a href="https://doi.org/10.1101/2023.07.17.549167"  class="external-link" target="_blank" rel="noopener">From Asymptomatic to Symptomatic: Multiomics profiling of the temporal response of grapevine viral-mixed infection</a></strong></p>
<p>




<div class="tabs tabs-left">
  




<style>
  .tabs input#tab-0-0:checked ~ .tab-content-0-0 {
    display: block;
  }
</style>

<input type="radio" class="tab-input" name="tab-select-0" id="tab-0-0" checked/>
<label for="tab-0-0" class="tab-label">Plus d&#39;informations</label>
<div class="tab-content tab-content-0-0">
  <strong>Statut :</strong> PrePrint<br>
<strong>Auteurs :</strong> M.L. Fall, <strong>A. Poursalavati</strong>üß¨, A. Sidib√©, D. Xu, P. Lemoyne, G.S. Martins, V.J. Javaran, P. Moffett, C. Odile<br>
<strong>Mots cl√©s :</strong> virome, RNAi profiling, seasonality of virus-like symptoms, transcriptomics, metabolomics, host-virome interaction<br>
<strong>Voir en ligne :</strong> <a href="https://doi.org/10.1101/2023.07.17.549167"  class="external-link" target="_blank" rel="noopener">bioRxiv</a>
</div>







<style>
  .tabs input#tab-0-1:checked ~ .tab-content-0-1 {
    display: block;
  }
</style>

<input type="radio" class="tab-input" name="tab-select-0" id="tab-0-1" />
<label for="tab-0-1" class="tab-label">R√©sum√©</label>
<div class="tab-content tab-content-0-1">
  <blockquote>
<p>Mixed viral infections are common in grapevines. However, our understanding of the factors and signaling pathways that influence the expression of viral symptoms in mixed infections is still incomplete. In a previous study, we revealed that the presence of grapevine leafroll-associated virus species in mixed infections was randomly associated with the development of virus-like symptoms. To understand what drives the timing of these virus-like symptoms in mixed infections, we used dsRNA and total RNA sequencing and metabolomic analysis to profile the viromes, metabolites, and transcripts of grapevine leaves collected at two different times of the year (summer and autumn). We demonstrated that neither viral titre nor virome composition changes were associated with symptom expression in autumn. The total phenolic content and antioxidant capacity increased in most plants except for those with early onset symptoms. According to the results of differential gene expression analysis, cell wall biosynthesis pathways were significantly downregulated in all grapevine plants infected with grapevine leafroll-associated virus 3, grapevine asteroid mosaic-associated virus, and grapevine Pinot gris virus. In addition, polyketide pathways were significantly upregulated in all cultivars, while flavonoid precursor (e.g. abscisic acid) production was significantly reduced in plants that expressed strong virus-like symptoms. In the &lsquo;Vidal&rsquo; cultivar, an uncharacterized double-stranded RNA-binding protein (DRB) appears to play a critical role in the plant&rsquo;s antiviral defences, supporting the recent hypothesis that DRBs make an important contribution to dominant antiviral responses in plants. The seasonality of the expression virus-like symptoms appears to be a consequence of the dynamic interactions between antiviral factors and viral counter-defences that occur at different developmental stages of grapevine.</p>
</blockquote>

</div>




</div>



<br>
<strong>2023 - <a href="https://doi.org/10.1101/2023.05.10.540251"  class="external-link" target="_blank" rel="noopener">dsRNA-based viromics: A novel tool unveiled hidden soil viral diversity and richness</a></strong></p>
<p>




<div class="tabs tabs-left">
  




<style>
  .tabs input#tab-1-0:checked ~ .tab-content-1-0 {
    display: block;
  }
</style>

<input type="radio" class="tab-input" name="tab-select-1" id="tab-1-0" checked/>
<label for="tab-1-0" class="tab-label">Plus d&#39;informations</label>
<div class="tab-content tab-content-1-0">
  <strong>Statut :</strong> PrePrint<br>
<strong>Auteurs :</strong> <strong>A. Poursalavati</strong>üß¨, A. Larafa, M.L. Fall<br>
<strong>Mots cl√©s :</strong> RNA viruses, soil virome, RdRps evolution, dsRNA, Viromics, Metaviromics<br>
<strong>Voir en ligne :</strong> <a href="https://doi.org/10.1101/2023.05.10.540251"  class="external-link" target="_blank" rel="noopener">bioRxiv</a>
</div>







<style>
  .tabs input#tab-1-1:checked ~ .tab-content-1-1 {
    display: block;
  }
</style>

<input type="radio" class="tab-input" name="tab-select-1" id="tab-1-1" />
<label for="tab-1-1" class="tab-label">R√©sum√©</label>
<div class="tab-content tab-content-1-1">
  <blockquote>
<p>Viruses play a crucial role in agroecosystem functioning. However, few studies have examined the diversity of the soil virome, especially when it comes to RNA viruses. Despite the great progress in viral metagenomics and metatranscriptomics (metaviromics) toward RNA viruses characterization, soil RNA viruses‚Äô ecology is embryonic compared to DNA viruses. We currently lack a wet lab. method to accurately unhide the true soil viral diversity. To overcome this limitation, we developed dsRNA-based methods capitalizing on our expertise in soil RNA extraction and dsRNA extraction ported from studies of phyllosphere viral diversity. This proposed method detected both RNA and DNA viruses and is proven to capture a greater soil virus diversity than existing methods, virion-associated nucleic enrichment, and metaviromics. Indeed, using this method we detected 284 novel RNA-dependent RNA polymerases and expanded the diversity of Birnaviridae and Retroviridae viral families to agricultural soil, which, to our knowledge, have never been reported in such ecosystem. The dsRNA-based method is cost-effective in terms of affordability and requirements for data processing, facilitating large-scale and high-throughput soil sample processing to unlock the potential of the soil virome and its impact on biogeochemical processes (e.g. carbon and nutrient cycling). This method can also benefit future studies of viruses in complex environments, for example, to characterize RNA viruses in the human gut or aquatic environment where RNA viruses are less studied mainly because of technical limitations.</p>
</blockquote>

</div>




</div>



<br>
<strong>2023 - <a href="https://doi.org/10.3389/fmicb.2023.1192781"  class="external-link" target="_blank" rel="noopener">NanoViromics: long-read sequencing of dsRNA for plant virus and viroid rapid detection</a></strong></p>
<p>




<div class="tabs tabs-left">
  




<style>
  .tabs input#tab-2-0:checked ~ .tab-content-2-0 {
    display: block;
  }
</style>

<input type="radio" class="tab-input" name="tab-select-2" id="tab-2-0" checked/>
<label for="tab-2-0" class="tab-label">Plus d&#39;informations</label>
<div class="tab-content tab-content-2-0">
  <strong>Statut :</strong> Published<br>
<strong>Auteurs :</strong> V.J. Javaran, <strong>A. Poursalavati</strong>üß¨, P. Lemoyne, D.T. Ste-Croix, P. Moffett, M.L. Fall<br>
<strong>Mots cl√©s :</strong> dsRNA, total RNAs, grapevine viruses, virus detection, viroid, nanopore sequencing, Illumina Miseq sequencing, plant pathology<br>
<strong>Voir en ligne :</strong> <a href="https://doi.org/10.3389/fmicb.2023.1192781"  class="external-link" target="_blank" rel="noopener">Frontiers in Microbiology</a>
</div>







<style>
  .tabs input#tab-2-1:checked ~ .tab-content-2-1 {
    display: block;
  }
</style>

<input type="radio" class="tab-input" name="tab-select-2" id="tab-2-1" />
<label for="tab-2-1" class="tab-label">R√©sum√©</label>
<div class="tab-content tab-content-2-1">
  <blockquote>
<p>There is a global need for identifying viral pathogens, as well as for providing certified clean plant materials, in order to limit the spread of viral diseases. A key component of management programs for viral-like diseases is having a diagnostic tool that is quick, reliable, inexpensive, and easy to use. We have developed and validated a dsRNA-based nanopore sequencing protocol as a reliable method for detecting viruses and viroids in grapevines. We compared our method, which we term direct-cDNA sequencing from dsRNA (dsRNAcD), to direct RNA sequencing from rRNA-depleted total RNA (rdTotalRNA), and found that it provided more viral reads from infected samples. Indeed, dsRNAcD was able to detect all of the viruses and viroids detected using Illumina MiSeq sequencing (dsRNA-MiSeq). Furthermore, dsRNAcD sequencing was also able to detect low-abundance viruses that rdTotalRNA sequencing failed to detect. Additionally, rdTotalRNA sequencing resulted in a false-positive viroid identification due to the misannotation of a host-driven read. Two taxonomic classification workflows, DIAMOND &amp; MEGAN (DIA &amp; MEG) and Centrifuge &amp; Recentrifuge (Cent &amp; Rec), were also evaluated for quick and accurate read classification. Although the results from both workflows were similar, we identified pros and cons for both workflows. Our study shows that dsRNAcD sequencing and the proposed data analysis workflows are suitable for consistent detection of viruses and viroids, particularly in grapevines where mixed viral infections are common.</p>
</blockquote>

</div>




</div>



<br>
<strong>2022 - <a href="https://doi.org/10.1094/PBIOMES-12-22-0108-TA"  class="external-link" target="_blank" rel="noopener">Soil Metatranscriptomics: An Improved RNA Extraction Method Toward Functional Analysis Using Nanopore Direct RNA Sequencing</a></strong></p>
<p>




<div class="tabs tabs-left">
  




<style>
  .tabs input#tab-3-0:checked ~ .tab-content-3-0 {
    display: block;
  }
</style>

<input type="radio" class="tab-input" name="tab-select-3" id="tab-3-0" checked/>
<label for="tab-3-0" class="tab-label">Plus d&#39;informations</label>
<div class="tab-content tab-content-3-0">
  <strong>Statut :</strong> Published<br>
<strong>Auteurs :</strong> <strong>A. Poursalavati</strong>üß¨, V.J. Javaran, I. Laforest-Lapointe, M.L. Fall<br>
<strong>Mots cl√©s :</strong> dsRNA, total RNAs, grapevine viruses, virus detection, viroid, nanopore sequencing, Illumina Miseq sequencing, plant pathology<br>
<strong>Voir en ligne :</strong> <a href="https://doi.org/10.1094/PBIOMES-12-22-0108-TA"  class="external-link" target="_blank" rel="noopener">Phytobiomes</a>
</div>







<style>
  .tabs input#tab-3-1:checked ~ .tab-content-3-1 {
    display: block;
  }
</style>

<input type="radio" class="tab-input" name="tab-select-3" id="tab-3-1" />
<label for="tab-3-1" class="tab-label">R√©sum√©</label>
<div class="tab-content tab-content-3-1">
  <blockquote>
<p>Soil microbes play an undeniable role in sustainable agriculture, plant health, and soil management. A deeper understanding of soil microbial composition and function has been gained through next-generation sequencing. Although soil metagenomics has provided valuable information about microbial diversity, issues stemming from RNA extraction, low RNA abundance in some microbial populations (e.g., viruses), and messenger RNA enrichment have slowed the progress of soil metatranscriptomics. A variety of soil RNA extraction methods have been developed thus far yet none of the available protocols can obtain RNA with high quality, purity, and yield for third-generation sequencing. The latter requires RNA with high quality and large quantities (with no or low contamination such as humic acids). Also, use of commercial kits for in-batch soil RNA extraction is quite expensive, and these commercial kits lack buffer composition details, which prevents the optimization of protocols for different soil types. An improved and cost-effective method for extracting RNAs from mineral and organic soils is presented in this article. An acidic sodium acetate buffer and phosphate buffer with modifications to bead beating and nucleic acid precipitation lead to higher RNA yields and quality. Using this method, we obtained almost DNA-free RNA. By using nanopore&rsquo;s direct RNA sequencing, the extracted contamination-free RNAs were successfully sequenced. Finally, taxonomic groups such as bacteria, fungi, archaea, and viruses were classified and profiled, and functional annotation of the datasets was carried out using an in-house customized bioinformatics workflow.</p>
</blockquote>

</div>




</div>



<br>
<strong>2021 - <a href="https://www.nature.com/articles/s41598-021-99066-6"  class="external-link" target="_blank" rel="noopener">Toward understanding of the methoxylated flavonoid biosynthesis pathway in <em>Dracocephalum kotschyi</em> Boiss.</a></strong></p>
<p>




<div class="tabs tabs-left">
  




<style>
  .tabs input#tab-4-0:checked ~ .tab-content-4-0 {
    display: block;
  }
</style>

<input type="radio" class="tab-input" name="tab-select-4" id="tab-4-0" checked/>
<label for="tab-4-0" class="tab-label">Plus d&#39;informations</label>
<div class="tab-content tab-content-4-0">
  <strong>Statut :</strong> Published<br>
<strong>Auteurs :</strong> <strong>A. Poursalavati</strong>üß¨, S. Rashidi-Monfared, A. Ebrahimi<br>
<strong>Mots cl√©s :</strong> RNA-Seq, Transcriptomics, Flavonoid Pathway, qRT-PCR, HPLC<br>
<strong>Voir en ligne :</strong> <a href="https://www.nature.com/articles/s41598-021-99066-6"  class="external-link" target="_blank" rel="noopener">Scientific Reports</a>
</div>







<style>
  .tabs input#tab-4-1:checked ~ .tab-content-4-1 {
    display: block;
  }
</style>

<input type="radio" class="tab-input" name="tab-select-4" id="tab-4-1" />
<label for="tab-4-1" class="tab-label">R√©sum√©</label>
<div class="tab-content tab-content-4-1">
  <blockquote>
<p>Nowadays, with the development and advancement of next-generation sequencing technologies, a new path has been provided for transcriptomic studies. In this study, the transcriptome of Dracocephalum kotschyi Boiss., as an endemic and endangered plant which is contained a large amount of valuable secondary metabolites with antioxidant and anticancer properties, was sequenced. Then functional annotation and gene ontology analysis for 165,597 assembled transcripts were performed, most were associated with the metabolic pathways. This might be because there are various active biochemical pathways in this plant. Furthermore, after comprehensive transcript annotation, the putative genes involved in the main metabolic pathways of D. kotschyi were identified. Then, the biosynthetic pathway of its valuable methoxylated flavones was proposed. Finally, the accumulations of important methoxylated-flavone metabolites in three different tissues were quantified by HPLC. The relative expression of the genes involved in the proposed pathway was investigated by qRT-PCR, which indicated high expression levels in the bud tissue. The present results may lead to the design strategies to preserve the genetic diversity of endangered D. kotschyi plants and apply the new methods for engineering its valuable methoxylated-flavones pathway.</p>
</blockquote>

</div>




</div>



<br>
<strong>2021 - <a href="https://doi.org/10.1007/s10681-021-02844-6"  class="external-link" target="_blank" rel="noopener">Population and individual multivariate analysis of white (<em>Morus alba</em>), red (<em>Morus rubra</em>) and black (<em>Morus nigra</em>) mulberry genotypes: applications for breeding, conservation and development</a></strong></p>
<p>




<div class="tabs tabs-left">
  




<style>
  .tabs input#tab-5-0:checked ~ .tab-content-5-0 {
    display: block;
  }
</style>

<input type="radio" class="tab-input" name="tab-select-5" id="tab-5-0" checked/>
<label for="tab-5-0" class="tab-label">Plus d&#39;informations</label>
<div class="tab-content tab-content-5-0">
  <strong>Statut :</strong> Published<br>
<strong>Auteurs :</strong> A. Ebrahimi, <strong>A. Poursalavati</strong>üß¨, M. Mohamadi-Esboei, S. Rashidi-Monfared, M. Sahebi, M. Amerian, H. Hassaneian-Khoshro<br>
<strong>Mots cl√©s :</strong> Mulberry, Multivariate analysis, Population analysis, Physiological and chemical traits<br>
<strong>Voir en ligne :</strong> <a href="https://doi.org/10.1007/s10681-021-02844-6"  class="external-link" target="_blank" rel="noopener">Euphytica</a>
</div>







<style>
  .tabs input#tab-5-1:checked ~ .tab-content-5-1 {
    display: block;
  }
</style>

<input type="radio" class="tab-input" name="tab-select-5" id="tab-5-1" />
<label for="tab-5-1" class="tab-label">R√©sum√©</label>
<div class="tab-content tab-content-5-1">
  <blockquote>
<p>Mulberry (Morus spp.) has always been a favorite plant of gardners, farmers and researchers because of its edible fruits, medicinal properties and timber. We have investigated the morphological, physiological and chemical variabilities of white (Morus alba), red (Morus rubra) and black (Morus nigra) mulberry genotypes grown in four regions of Iran. Among the genotypes assessed, fresh fruit weight varied between 3.2 and 87.9 g, total chlorophyll ranged from 4.3 to 13.4 mg/g fresh weight and vitamin C content ranged from 5.5 to 64 mg per 100 g. The highest and lowest antioxidant activity was 275 and 1575 mg ascorbic acid equivalent (AAE) 100 g‚àí1 fresh weight, respectively. The population analysis indicated that the Gorgan population, in terms of morphological and physiological traits, and Shahrood population, in terms of chemical traits, had the highest coefficient of variation. Correlation analysis showed that fruit size and taste were positively correlated with foliar traits and that phenolic content was positively correlated with both antioxidant activity and total soluble solids (TSS). Of the indices assessed, the Berger‚ÄìParker index was the most valuable because it was able to identify significant differences between populations in terms of the studied traits. The results of this study confirm that Iran is one of the mulberry phenotypic and biochemical diversity areas and provide information of possible use for the development, conservation and selection of parents for cross-combinations in breeding programs.</p>
</blockquote>

</div>




</div>



<br>
<strong>2019 - <a href="https://cell.ijbio.ir/article_1593.html?lang=en"  class="external-link" target="_blank" rel="noopener">The Comparison of Assembly Softwares and Gene Ontology Analysis using transcriptome sequencing data from <em>Dracocephalum kotschyi</em> Boiss.</a></strong></p>





<div class="tabs tabs-left">
  




<style>
  .tabs input#tab-6-0:checked ~ .tab-content-6-0 {
    display: block;
  }
</style>

<input type="radio" class="tab-input" name="tab-select-6" id="tab-6-0" checked/>
<label for="tab-6-0" class="tab-label">Plus d&#39;informations</label>
<div class="tab-content tab-content-6-0">
  <strong>Statut :</strong> Published<br>
<strong>Auteurs :</strong> <strong>A. Poursalavati</strong>üß¨, A. Ebrahimi, S. Rashidi-Monfared<br>
<strong>Mots cl√©s :</strong> Trinity, SOAPdenovo-Trans, Oases-velvet, Trans-ABySS, Gene Ontology<br>
<strong>Voir en ligne :</strong> <a href="https://cell.ijbio.ir/article_1593.html?lang=en"  class="external-link" target="_blank" rel="noopener">Cellular and Molecular Research (Iranian Journal of Biology)</a>
</div>







<style>
  .tabs input#tab-6-1:checked ~ .tab-content-6-1 {
    display: block;
  }
</style>

<input type="radio" class="tab-input" name="tab-select-6" id="tab-6-1" />
<label for="tab-6-1" class="tab-label">R√©sum√©</label>
<div class="tab-content tab-content-6-1">
  <blockquote>
<p>With fast advances in next generation sequencing technologies, they has become powerful and low-cost tools for transcriptome studies, Nowadays; de novo assembly of transcriptome data, has caused the formation of the new pathway in the study of non-model genome species. With the expansion of this technology and increasing the number of assembly softwares, choosing the best software for assembling transcriptome sequencing data has become a challenge for biologists. For the first time in this study, we used transcriptome sequencing data of Dracocephalum kotschyi in order to reach the appropriate parameters and superior software; so here we used Oases-velvet, SOAPdenovo-Trans, Trans-ABySS and Trinity softwares with two different values of K parameter; K=25 and K=32. The results of assembly by each software were compared to others in the term of several criteria. The result suggests the superiority of Trinity and Trans-ABySS softwares. Finally, the output of the best assembly was used to estimate abundance of various isoforms and Gene Ontology analysis as regards to the pharmaceutical properties of this plant and the high amount of secondary metabolites, the highest frequency of sections in the biological processes was related to the metabolic activity.</p>
</blockquote>

</div>




</div>



<ul>
<li>Projets</li>
<li>Th√®ses</li>
<li>Ateliers, S√©minaires, Pr√©sentations<br>
<strong>Davantage de d√©tails seront ajout√©s bient√¥t !</strong></li>
</ul>
  </article>
</section>

  

    </div>

    <footer class="footer">
  <section class="container">
    ¬©
    
      2019 -
    
    2024
     Naser Poursalavati 
    ¬∑
    
    Propuls√© par <a href="https://gohugo.io/" target="_blank" rel="noopener">Hugo</a> & <a href="https://github.com/luizdepra/hugo-coder/" target="_blank" rel="noopener">Coder</a>.
    
  </section>
</footer>

  </main>

  

  
  
  <script src="/js/coder.min.6ae284be93d2d19dad1f02b0039508d9aab3180a12a06dcc71b0b0ef7825a317.js" integrity="sha256-auKEvpPS0Z2tHwKwA5UI2aqzGAoSoG3McbCw73gloxc="></script>
  

  

  


  

  

  

  

  

  

  

  

  

  

  

  

  
</body>

</html>